swap/swap.go

```diff
@@ -474,12 +474,14 @@ func cashCheque(s *Swap, otherSwap contract.Contract, opts *bind.TransactOpts, c
 		return
 	}
 
+	metrics.GetOrRegisterCounter("swap.cheques.cashed.honey", nil).Inc(result.TotalPayout.Int64())
 	if result.Bounced {
 		swapLog.Warn("cheque bounced", "tx", receipt.TxHash)
 		return
 		// TODO: do something here
 	}
 
+	metrics.GetOrRegisterCounter("swap.cheques.cashed.bounced", nil).Inc(1)
```

Comment:
This seems to be in the wrong place. should be on line 479.

Reply:
hahaha right, thanks